### PR TITLE
Add help and drawing dh

### PR DESCRIPTION
**Pull request by dougstarfish**
add explicit --help or -h flag to show usage
add ability to change border box width around nodes
add a sample ged2dotrc.sample
add the beginnings of a test for nosex.ged, but unsure how to finish it and link it in to assert true on failure

### CODE REVIEW

**Comment by dougstarfish** [comment]
Have you thought at all about a mechanism where all siblings could be eliminated and just have the tree? This would be for the purpose of showing lineage, and would allow many generations to be displayed since once you get beyond ~5 generations, there's a lot of thinning that happens naturally (especially on the mother's side where family name is unknown many times prior to about 1850)

**Comment by vmiklos** [comment]
Can you please rebase (`git pull -r && git push -f ...`) in the hope that fixes the CI failure? I don't want to fix it up for you, like I did it yesterday. :-)

> Have you thought at all about a mechanism where all siblings could be eliminated and just have the tree?

There is already layoutMaxSiblingDepth, did you find that? If you set that to 0, you may get what you want.

**Comment by dougstarfish** [comment]
layoutMaxSiblingDepth did not do it. I tried that and it wasn't clear what it affect it had, but it didn't seem to remove siblings as far as I could tell (or at least not all of them). I'm trying to figure out the rebase part, but I'm not a git expert and I'm stuck at the push.

**Comment by dougstarfish** [comment]
I did a new minor change to TODO and a push, but still the CI failure. (I ended up re-checkout of my branch and a tiny commit). I don't know why the CI fails since make check works just fine in my branch.

File: ged2dot.py [hunk 1]
```diff
@@ -840,7 +840,6 @@ def load(self) -> None:
                         elif self.inDeat:
                             self.indi.deat = year
 
-            # pylint: disable=broad-except
```
Comment (vmiklos) [inline ged2dot.py]: Don't remove this, it was there on purpose to document that catching `Exception` is OK in this particular case, even if it's a bad idea in general.

File: test/test.py [hunk 2]
```diff
@@ -50,6 +50,16 @@ def test_partialname(self) -> None:
         assert indi
         self.assertTrue("None" not in indi.getLabel())
 
+    def test_nosex(self) -> None:
+        # if there is no sex, this should fail and indicate line number
+        configDict = {
```
Comment (vmiklos) [inline test/test.py]: If this is a work in progress test, then disable it, so `make check` keeps passing after accepting this PR.

**Comment by vmiklos** [comment]
Regarding the test itself, I think all we need is to mock sys.exit, I'll look at that later. Thanks for the patches!

File: ged2dot.py (hunk 1 above)
Comment (dougstarfish) [reply ged2dot.py]: oops. That was purely unintentional. Not sure how I did that.

**Pull request merged**